hackathon_site/dashboard/frontend/src/components/general/Footer/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import styles from "./Footer.module.scss";
+
+const Footer = () => {
+    let date = new Date();
+
+    if (date.getFullYear() === 2020) {
+        var outputMessage =
```

Comment:
Stick to using ES6 `let` or `const` instead of `var`. A simpler way of writing this would be:
```javascript
const yearRange = date.getFullYear() === 2020 ? "" : " - " + date.getFullYear();
const outputMessage = "© 2020" + yearRange + " built by the web team at MakeUofT (University of Toronto)";
```

Reply:
I was looking into it, so `let` and `const ` doesn't allow you to change its values? and that's why you wanted to use those two instead `var` ?

Reply:
`const` doesn't let you modify values ("constant" type). Most of the time `const` is what you'll want to use for components etc. `let` does, and in most cases behaves similarly to `var`, just more precise in what scopes it lives at (block scope). Some ES6 linters will also complain about the user of `var`, so we should avoid using it.

Here's a documentation link on `let`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let